scripts/zones/Mhaura/npcs/Laughing_Bison.lua

```diff
@@ -12,7 +12,30 @@ function onTrade(player,npc,trade)
 end;
 
 function onTrigger(player,npc)
-    player:startEvent(333);
+    -- Each boat comes every 1152 seconds/8 game hours, 4 hour offset between Selbina and Aht Urghan
+    -- Original timer: local timer = 1152 - ((os.time() - 1009810584)%1152);
+    local timer = 1152 - ((os.time() - 1009810802)%1152);
+    local destination = 0; -- Selbina, set to 1 for Al Zhabi
+    local direction = 0; -- Arrive, 1 for depart
+    local waiting = 216; -- Offset for Selbina
+
+   -- Next ferry is Al Zhabi for higher values.
+    if (timer >= 576) then
+      destination = 1;
+      timer = timer - 576;
+      waiting = 193;
+    else if (destination == 0) then
```

Comment:
`elseif` not `else if`. just oen of those gotcha's when you've been a C-variant coder and then you try lua.

You still have tabs and indent probs. 4 spaces per indentation level.

```
elseif condition then
    stuff
    morestuff
    if moreConditions then
        morestuff
    end
end
```

Reply:
appreciate the insight!